factory/formatter.ts

```diff
@@ -25,8 +25,11 @@ import { UndefinedTypeFormatter } from "../src/TypeFormatter/UndefinedTypeFormat
 import { UnionTypeFormatter } from "../src/TypeFormatter/UnionTypeFormatter";
 import { UnknownTypeFormatter } from "../src/TypeFormatter/UnknownTypeFormatter";
 import { VoidTypeFormatter } from "../src/TypeFormatter/VoidTypeFormatter";
+import { MutableTypeFormatter } from "../src/MutableTypeFormatter";
 
-export function createFormatter(config: Config): TypeFormatter {
+export type FormatterConfigurator = (formatter: MutableTypeFormatter) => void;
+
+export function createFormatter(config: Config, configurator?: FormatterConfigurator): TypeFormatter {
```

Comment:
Can you come up with a name for configurator that is easier to understand for someone who is new to the API?